pkg/plugins/platform/local.go

```diff
@@ -812,13 +812,26 @@ func (l *LocalApp) Config() error {
 func (l *LocalApp) Down(removeData bool) error {
 	l.DockerEnv()
 	settingsFilePath := l.AppConfig.SiteSettingsPath
+	_, missingConfig := CheckForConf(l.AppRoot())
 
-	err := dockerutil.ComposeCmd(l.ComposeFiles(), "down", "-v")
-	if err != nil {
-		util.Warning("Could not stop site with docker-compose. Attempting manual cleanup.")
-		err = Cleanup(l)
+	if !fileutil.FileExists(l.AppRoot()) {
+		err := Cleanup(l)
+		if err != nil {
+			util.Failed("Failed to remove %s: %s", l.GetName(), err)
```

Comment:
This results in a really verbose message when running `ddev rm` in a directory where a config doesn't exist, example:

"Failed to remove: Could not find a site in /Users/beeradb/Projects/go/src/github.com/drud/ddev. Please specify a site name or change directories: no .ddev/config.yaml file was found in this directory or any parent"

It feels like we could simplify that a lot.

Also, "%v" is the preferred Printf token for errors.